mmengine/model/base_module.py

```diff
@@ -166,6 +166,8 @@ def _dump_init_info(self):
         # dump the information to the logger file if there is a `FileHandler`
         for handler in logger.handlers:
             if isinstance(handler, FileHandler):
+                if (handler.stream is None):
```

Comment:
I also can not reproduce the error you met.

Reply:
I am analyzing the path of this issue, in my environment the FileHandler is closed after `import mlflow`. 
Can you tell me your `mlflow` version, mine is `2.1.1` 

(I think after import mlflow, I'm thinking about whether to open it again after checking if the log's file handler is closed.)

``` python
class MLflowVisBackend(BaseVisBackend): 
    def _init_env(...): 
        import mlflow 
```

Reply:
@zhouzaida 
I have committed a fix to `FileHandler` this issue.

Reply:
> @zhouzaida I have committed a fix to `FileHandler` this issue.

OK, thanks for your contribution.